### ISSUE TITLE
Katib experiment failing with  metric_logs:{time_stamp:"0001-01-01T00:00:00Z" metric:{name:"accuracy" value:"unavailable"}}

### ISSUE THREAD

**Issue by abijith-pradeep** [kubeflow/katib#2554]
### What happened?

I tried creating a random search experiment on the Katib UI within Kubeflow's Central Dashboard.  I was using the trial templates given in the UI with the two parameters lr (Learning Rate) and Momentum. When creating a new experiment it starts off with num-layers, lr and optimizer as the hyper-parameters. For running the experiment I had to delete the two hyper-parameters optimizer and num-layers and add the momentum parameter with the appropriate range (0.1 to 0.99) mentioned in the kubeflow-katib image provided in the trial-templates configmap. The experiment runs with 5 trials with all of the trials failing because the metrics Collector is unable to parse the collected metrics. Here are the logs for one of the trials:

![Image](https://github.com/user-attachments/assets/85bf1e20-5da0-4571-bbd8-699c6a415e3a)

This doesn't happen when running a katib experiment through the Python SDK. Any help would be greatly appreciated

### What did you expect to happen?

The experiment completes with the best trial parameters printed out.

### Environment

Kubernetes version:
```bash
$ kubectl version
Client Version: v1.32.3+rke2r1
Kustomize Version: v5.5.0
Server Version: v1.32.3+rke2r1
```
Katib controller version:
```bash
$ kubectl get pods -n kubeflow -l katib.kubeflow.org/component=controller -o jsonpath="{.items[*].spec.containers[*].image}"
ghcr.io/kubeflow/katib/katib-controller:v0.18.0
```
Katib Python SDK version:
```bash
$ pip show kubeflow-katib
Name: kubeflow-katib
Version: 0.18.0
Summary: Katib Python SDK for APIVersion v1beta1
Home-page: https://github.com/kubeflow/katib/tree/master/sdk/python/v1beta1
Author: Kubeflow Authors
Author-email: premnath.vel@gmail.com
License: Apache License Version 2.0
Location: /opt/conda/lib/python3.11/site-packages
Requires: certifi, grpcio, kubeflow-training, kubernetes, protobuf, setuptools, six, urllib3
Required-by: 
```

### Impacted by this bug?

Give it a 👍 We prioritize the issues with most 👍
